e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js

```diff
@@ -577,6 +577,45 @@ describe("scenarios > visualizations > line chart", () => {
       cy.get(".y-axis-label").invoke("text").should("eq", "Average of Price");
     });
   });
+
+  describe("brush filters", () => {
+    it("should apply filters to the series selecting area range", () => {
+      cy.viewport(1280, 800);
+
+      visitQuestionAdhoc({
+        dataset_query: testQuery,
+        display: "line",
+      });
+
+      cy.get(".Visualization")
+        .trigger("mousedown", 100, 200)
+        .trigger("mousemove", 230, 200)
+        .trigger("mouseup", 230, 200);
+
+      cy.wait("@dataset");
+
+      cy.findByTestId("filter-pill").should(
+        "have.text",
+        "Created At is Apr 1, 12:00 AM – Sep 1, 2022, 12:00 AM",
+      );
+
+      cy.get(".Visualization .dot").should("have.length", 6);
+    });
+
+    it("should apply filter to the series after drilling down", () => {
```

Comment:
Technically this is not a brush filter - it's a regular drill

Reply:
should I keep it or it's better to remove it? I can add `(drill)` in the name

Reply:
removed this test, I'll add another test where x-axis is number in a different PR